src/app/digital/models/ioobjects/other/IC.ts

```diff
@@ -97,7 +97,7 @@ export class IC extends DigitalComponent {
     }
 
     public getData(): ICData {
-        return this.data;
+        return ICData.Create(this.collection)!;
```

Comment:
What was the reason for this again?
It scares me greatly
If you need to copy the ICData at some point, the copying should probably be done in the ICViewer somehow